composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "drupal/500px",
+  "name": "drupal/d500px",
```

Comment:
`$ composer require drupal/500px:1.x-dev
...
Your requirements could not be resolved to an installable set of packages.
Problem 1
    - The requested package drupal/500px could not be found in any version, there may be a typo in the package name.`

`$ composer require drupal/d500px:1.x-dev` => OK